public/query.js

```diff
@@ -3,7 +3,7 @@ const urlParams = new URLSearchParams(window.location.search);
 let id = urlParams.get('id');
```

Comment:
If there is no id in the url, does the id get set to undefined or null on line 3?

Reply:
If there is no id in the url, it gets set to null on line 3, and then if it's null on line 5, it'll set it to a random number of 1-100